src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const population = [...document.querySelectorAll('.population')];
+
+const num = Array.from(population).map((ele) => {
+  const txt = ele.textContent.replace(/,/g, '');
+
+  return +txt;
+});
```

Comment:
The use of `Array.from(population)` is correct, but since `population` is already an array (created using the spread operator), you can directly use `population.map(...)` instead of `Array.from(population).map(...)`. This will make the code slightly more efficient.